tibanna_ffcommon/generic_qc_utils.py

```diff
@@ -0,0 +1,47 @@
+from .exceptions import (
+    GenericQcException
+)
+
+def check_qc_workflow_args(input_file_args, generic_qc_args):
+    """
+    This function performs basic sanity checks on the QC and input files
+    """
```

Comment:
For new code I prefer type annotations and more info in the docstring on structure etc - PyCharm will generate these for you quickly with minimal overhead. I won't comment this everywhere but it applies in several places.

Reply:
I added type hints and more detailed docstrings to all new functions